client/protocols/hinting.py

```diff
@@ -145,7 +146,7 @@ def run(self, messages):
                     print("\n{}".format(hint.rstrip()))
 
                     if post_prompt:
-                        prompt_user(post_prompt, hint_info)
+                        results['prompts'][query] = prompt.explanation_msg(post_prompt)
```

Comment:
Just factoring out this code into a util method. This protocol is only run on certain Scheme assignments.